packages/user-operation-controller/src/utils/validation.test.ts

```diff
@@ -16,16 +25,17 @@ const ADD_USER_OPERATION_REQUEST_MOCK = {
   value: '0x3',
   maxFeePerGas: '0x4',
   maxPriorityFeePerGas: '0x5',
-};
+} as AddUserOperationRequest;
 
 const ADD_USER_OPERATION_OPTIONS_MOCK = {
-  chainId: '0x1',
+  networkClientId: 'testNetworkClientId',
+  origin: 'test.com',
   smartContractAccount: {
     prepareUserOperation: jest.fn(),
     updateUserOperation: jest.fn(),
     signUserOperation: jest.fn(),
   },
-};
+} as AddUserOperationOptions;
 
```

Comment:
Unless there's a good reason using a cast here and elsewhere, I would use type checking at assignment as it is more readable and a stronger type assertion than casting.
```suggestion
const ADD_USER_OPERATION_OPTIONS_MOCK: AddUserOperationOptions = {
  networkClientId: 'testNetworkClientId',
  origin: 'test.com',
  smartContractAccount: {
    prepareUserOperation: jest.fn(),
    updateUserOperation: jest.fn(),
    signUserOperation: jest.fn(),
  },
};

```

Reply:
Definitely agree, I think I just got into the habit of casting for the partial cases so a welcome surprise when we have a full object and don't need to 😄